README.md

```diff
@@ -1,6 +1,7 @@
-# docker101
+# Docker101
 
 Docker Introduction: <https://docs.microsoft.com/en-us/dotnet/architecture/microservices/container-docker-introduction>
```

Comment:
this is a public repo and that's an internal link - probably makes sense to put an "MS Internal" qualifier on the link